src/goto-instrument/code_contracts.cpp

```diff
@@ -213,12 +225,10 @@ void code_contractst::add_quantified_variable(
   else if(expression.id() == ID_exists || expression.id() == ID_forall)
   {
     // When a quantifier expression is found,
-    // 1. get quantified variables
+    // for each quantified variable ...
```

Comment:
If you decide to remove `1.`, update the following comments too; otherwise, leave it here.

Reply:
Done, as suggested.